diff --git a/Makefile b/Makefile
index dc38a36..1b26472 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,9 @@
 CC = gcc
 
-CFLAGS    = -Wall -Wextra -std=gnu99 -pedantic -g -DGDK_VERSION_MIN_REQUIRED=GDK_VERSION_3_4
+PKGDATADIR ?= .
+
+CFLAGS    = -Wall -Wextra -std=gnu99 -pedantic -g -DGDK_VERSION_MIN_REQUIRED=GDK_VERSION_3_4 \
+            -DPKGDATADIR=\"$(PKGDATADIR)\"
 GTKCFLAGS = `pkg-config --cflags gtk+-3.0`
 GTKLIBS   = `pkg-config --libs gtk+-3.0`
 
diff --git a/gui.c b/gui.c
index 63a0d17..9fbfd5d 100644
--- a/gui.c
+++ b/gui.c
@@ -12,8 +12,8 @@ void createGUI() {
   GtkBuilder *builder;
 
   builder = gtk_builder_new();
-  gtk_builder_add_from_file(builder, "slowrx.ui",      NULL);
-  gtk_builder_add_from_file(builder, "aboutdialog.ui", NULL);
+  gtk_builder_add_from_file(builder, PKGDATADIR "/slowrx.ui",      NULL);
+  gtk_builder_add_from_file(builder, PKGDATADIR "/aboutdialog.ui", NULL);
   
   gui.button_abort    = GTK_WIDGET(gtk_builder_get_object(builder,"button_abort"));
   gui.button_browse   = GTK_WIDGET(gtk_builder_get_object(builder,"button_browse"));
